static/js/subs.js

```diff
@@ -148,6 +148,20 @@ exports.update_stream_description = function (sub, description, rendered_descrip
     stream_edit.update_stream_description(sub);
 };
 
+exports.update_check_button_for_sub = function (sub) {
+    var button = check_button_for_sub(sub);
+    if (sub.subscribed) {
+        button.addClass("checked");
+    } else {
+        button.removeClass("checked");
+    }
+    if (sub.should_display_subscription_button) {
+        button.removeClass("disabled");
+    } else {
+        button.addClass("disabled");
+    }
```

Comment:
I tried it this way:
`   
    sub.subscribed == true ? button.addClass("checked") : button.removeClass("checked");
    sub.should_display_subscription_button == true ? button.removeClass("disabled") : button.addClass("disabled");
`
But that returns errors in `./tests/lint`.  Is one line `if.. else` is not allowed?